test/broadcast.jl

```diff
@@ -274,6 +274,12 @@ let x = [1,2,3]
     @test f(x) == [1,4,9]
 end
 
+# Issue #23622: @. with chained comparisons
+let x = [1,2,3]
+    @test (1 .< x .< 3) == @.(1 < x < 3) == [false, true, false]
```

Comment:
Add `@.(1 .< x .< 3)` too, to check for double-dotting bugs.

Reply:
Thanks! Updated.